2019/01.md

```diff
@@ -38,6 +38,7 @@ Deadline for advancement eligibility: [**January 19, 2019, 18:00 UTC**](https://
     1. Such proposals *should* include supporting materials when possible.
 1. Proposals looking to advance to stages 2, 3, or 4 must be added (and noted as such) *along with the necessary materials* prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
     1. Such proposals *must* link to supporting materials prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline. If these materials change substantially after the deadline, the proposal may be disqualified from advancement, based on the committee’s judgment.
+    1. Proposals looking to advance to stage 4 *must* link to a pull request into [the spec](https://github.com/tc39/ecma262), since the [process document](https://tc39.github.io/process-document/) requires one.
```

Comment:
Nit: it's the *process* that requires the PR, the document just wrote down the process. I would remove the word "document" here.